aws/resource_aws_api_gateway_domain_name_test.go

```diff
@@ -24,12 +25,12 @@ func TestAccAWSAPIGatewayDomainName_CertificateArn(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:          func() { testAccPreCheck(t); testAccPreCheckApigatewayEdgeDomainName(t) },
 		ProviderFactories: testAccProviderFactories,
-		CheckDestroy:      testAccCheckAWSAPIGatewayDomainNameDestroy,
+		CheckDestroy:      testAccCheckAWSAPIGatewayEdgeDomainNameDestroy,
```

Comment:
aside: maybe updating the test name and config could be helpful too when going back to understand why these use `Edge` related funcs at first glance --  `TestAccAWSAPIGatewayDomainName_EdgeCertificateArn` and `testAccAWSAPIGatewayDomainNameConfig_EdgeCertificateArn`

Reply:
Happy to use whatever convention you all feel is appropriate. The current test naming is based on the root attribute being tested so `certificate_arn` (which requires `endpoint_configuration` `types` `EDGE` but theoretically could be named `_EndpointConfiguration_Types_Edge`) versus `regional_certificate_arn` (which requires `endpoint_configuration` `types` `REGIONAL` but theoretically could be named `_EndpointConfiguration_Types_Regional`) etc.

Reply:
If you have continuing feels on this, please create a new issue and let's discuss!